sdk/automation/Azure.ResourceManager.Automation/src/Generated/AutomationAccountResource.cs

```diff
@@ -1601,14 +1601,14 @@ public virtual Response<LinkedWorkspace> GetLinkedWorkspace(CancellationToken ca
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentException"> <paramref name="typeName"/> is an empty string, and was expected to be non-empty. </exception>
         /// <exception cref="ArgumentNullException"> <paramref name="typeName"/> is null. </exception>
-        /// <returns> An async collection of <see cref="TypeField" /> that may take multiple service requests to iterate over. </returns>
-        public virtual AsyncPageable<TypeField> GetFieldsByTypeObjectDataTypesAsync(string typeName, CancellationToken cancellationToken = default)
+        /// <returns> An async collection of <see cref="AutomationModuleField" /> that may take multiple service requests to iterate over. </returns>
+        public virtual AsyncPageable<AutomationModuleField> GetFieldsByTypeAsync(string typeName, CancellationToken cancellationToken = default)
```

Comment:
Maybe `GetFieldsByObjectDataType` is a better method name.

Reply:
This is a shared function for both `AutomationAccountResource` and `AutomationAccountModuleResource`, that in `AutomationAccountModuleResource` there is another method return the same value but different scope, one for module one for account.
Related to your first comment, so leave it as is here.